tests/phpunit/tests/st/strategy/shortcode/test-wpml-pb-shortcode-content-wrapper.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * @group page-builders-shortcodes
+ * @group wpmltm-2970
+ */
+class Test_WPML_PB_Shortcode_Content_Wrapper extends WPML_PB_TestCase {
+
+	/**
+	 * @test
+	 * @dataProvider dp_get_wrapped_content
+	 *
+	 * @param string $content
+	 * @param string $expected_content
+	 */
+	public function it_should_get_wrapped_content( $content, $expected_content ) {
+		$valid_shortcodes = array(
+			'av_button',
+			'av_notification',
+			'shortcode_nested',
+			'shortcode_A',
+		);
+
+		$subject = new WPML_PB_Shortcode_Content_Wrapper( $content, $valid_shortcodes );
+		$this->assertEquals( $expected_content, $subject->get_wrapped_content() );
+	}
+
+	public function dp_get_wrapped_content() {
+		return array(
```

Comment:
@strategio This is very hard to read and understand. To add extra test cases is messy and makes in harder to understand.

Can we change this to load source html and parsed html from files in a fixtures directory under the tests?

eg.

basic_test.html file
```html
This is the main text.

[av_button label='Click me' link='manually,http://google.com']
```

basic_test_parsed.html file
```html
[wpml_string_wrapper]This is the main text.[/wpml_string_wrapper]

[av_button label='Click me' link='manually,http://google.com']
```

Reply:
Done @brucepearson